pootle/apps/virtualfolder/helpers.py

```diff
@@ -46,3 +46,11 @@ def extract_vfolder_from_path(request_path):
         return None, request_path
     else:
         return vfti.vfolder, vfti.directory.pootle_path
+
+
+def vftis_for_children(directory):
+    """
+    Returns the vfoldertreeitems for a directory's children
+    """
+    child_pks = [child.pk for child in directory.children]
+    return VirtualFolderTreeItem.objects.filter(directory__pk__in=child_pks)
```

Comment:
A directory can have children (stores and other dirs) but if they are in a `VirtualFolderTreeItem` (or several of them), those vftis will have it directory to be the parent directory for those children. So why not just use `directory.vf_treeitems.all()`?